TROUBLESHOOTING.md

```diff
@@ -1,71 +1,73 @@
 
-### Success
-A successful startup for output to New Relic will show messages like this:
+(**TODO**: produce this condition and note what it looks like.)
 
-```
-   2023-07-21T16:04:30.85-0700 [CELL/0] OUT Downloaded droplet (28.6M)
-   2023-07-21T16:04:30.99-0700 [APP/PROC/WEB/0] ERR Fluent Bit v2.1.7
-   [ ... ]
-   2023-07-21T16:04:30.99-0700 [APP/PROC/WEB/0] ERR [2023/07/21 23:04:30] [ info] [input:dummy:dummy.0] initializing
-   2023-07-21T16:04:30.99-0700 [APP/PROC/WEB/0] ERR [2023/07/21 23:04:30] [ info] [input:dummy:dummy.0] storage_strategy='memory' (memory only)
-   2023-07-21T16:04:31.00-0700 [APP/PROC/WEB/0] ERR [2023/07/21 23:04:30] [ info] [output:nrlogs:nrlogs.0] configured, hostname=gov-log-api.newrelic.com:443
-   2023-07-21T16:04:31.00-0700 [APP/PROC/WEB/0] ERR [2023/07/21 23:04:30] [ info] [sp] stream processor started
-   2023-07-21T16:04:32.94-0700 [APP/PROC/WEB/0] ERR [2023/07/21 23:04:32] [ info] [output:nrlogs:nrlogs.0] gov-log-api.newrelic.com:443, HTTP status=202
-   2023-07-21T16:04:32.94-0700 [APP/PROC/WEB/0] ERR {"requestId":"8f7fe1fb-0001-bdd4-0651-01897ab29b7d"}
-   2023-07-21T16:04:33.22-0700 [CELL/SSHD/0] OUT Exit status 0
-```
+### Finding the logs in New Relic's UI
+New Relic has *Logs* in two places in its UI. There is a global *Logs* section, which is where you should look first. The URL of this view will start with https://one.newrelic.com/logger (in fact, if you are logged in, that URL should take you there) and the heading will be something like _All logs_. There is another, very similar looking, logs view in the *APM & Services* section under each entity New Relic knows about.
 
-Specifically, you should see:
-* `[output:nrlogs:nrlogs.0] gov-log-api.newrelic.com:443, HTTP status=202`
+Each APM & Service instance that New Relic classifies as a "service" will have its own individual logs that are available when navigating to a specific service. When navigating to the "Global Logs Inbox" as specified above, you will see all logs from all services. To filter on specific logs that are coming from `cg-logshipper` you can add a column of `newrelic.source`, or, run a query with the following: `newrelic.source:"api.logs"`. Logshipper will have a source of `newrelic.source:"api.logs"`, where application logs will have a source of `newrelic.source:"logs.APM"`.
+Which helped highlight that New Relic was in fact, receiving logs from the `cg-logshipper` application, along with adding a column of `newrelic.source`. If you plan to have the logshipper deployed in multiple environments, it may also be beneficial to add a column of `tags.space_name` so that when all instances of logshipper, as well as application instances are putting logs into the inbox, it is easier to see which environments are, in fact, sending logs to new relic.
```

Comment:
```suggestion
If you plan to have the logshipper deployed in multiple environments, it may also be beneficial to add a column of `tags.space_name` so that when all instances of logshipper as well as application instances are putting logs into the inbox, it is easier to see which environments are, in fact, sending logs to New Relic.
```